src/test/suite/DockerTestUtils.ts

```diff
@@ -117,13 +117,9 @@ export class DockerTestUtils {
    * Checks the status of the specified container and the availability of the database within it.
    *
    * @param containerName - The name of the container to check.
-   * @param dbExecutable - The executable for the database client.
    * @returns A Promise that resolves when the container status and database availability are checked.
    */
-  static async checkContainerStatus(
-    containerName: string = "mariadb",
-    dbExecutable: string = "mysql-client"
```

Comment:
wasn't this used somewhere else? Is it ok, just to delete it?

Reply:
It was not used anywhere else, therefore I deleted it.